tools/run_remote_benchmark.sh

```diff
@@ -85,6 +85,26 @@ case "$BENCHMARK_STRATEGY" in
   bash -x "${SCRIPT_PATH}/qdrant_collect_stats.sh" "$SERVER_CONTAINER_NAME"
   ;;
 
+  "parallel")
+  echo "Parallel benchmark, run upload&search at the same time"
+
+  SERVER_CONTAINER_NAME=${SERVER_CONTAINER_NAME:-"qdrant-continuous-benchmarks-with-volume"}
+
+  bash -x "${SCRIPT_PATH}/run_server_container_with_volume.sh" "$SERVER_CONTAINER_NAME"
```

Comment:
why RAM limit the start but after init? It should be consistent imo

Reply:
the actual RAM limitation is actually the same here, as default value for the 2nd arg (aka CONTAINER_MEM_LIMIT) in run_server_container_with_volume.sh is "25Gb". Bash is so bash :) I'll add "25Gb" to this line for simplicity.